lib/data/content/home/forewords.yml

```diff
@@ -0,0 +1,47 @@
+id: forewords
+title: Forewords
+content:
+  -
+    id: inger
+    img_src: /content/images/profile/inger-andersen.jpg
```

Comment:
Don't worry about changing it, but I think the party line is that we shouldn't include anything that wouldn't be translated in the yml file. Perhaps worth a chat with other developers to see where they stand. This is what Stacy (and Ferdi) used to tell me. Yes, I'm invoking the spirit of Ferdi on this one. ;)

Reply:
Not sure about this one @WKocur  - is this for the values of `id` and `img_src`?

Would the alternative to be to store them in a seperate file and reference them as variables?

Reply:
Yeah exactly - I guess the alternative is that this goes in a helper and then there's a method that retrieves the translations using I18n and merges them with the non-translatable config. 

A bit of a faff to code though... As far as I can tell:

Pros:
- Translations are only handling translations
- Easy to pass these files to translators and they don't have to work out what's config and what needs translating

Cons:
- Extra code
- Takes longer to write
- Separation of related information into different files

Like I say, too much effort to refactor now, but perhaps worth bearing in mind in the future and discussing with other devs :)